packages/web/src/components/composer-input/ComposerInput.tsx

```diff
@@ -58,13 +58,20 @@ const ComposerText = ({
   )
 }
 
-const createTextSections = (text: string) => {
+const createTextSections = (
+  text: string,
+  lineIndex: number
+): [JSX.Element[], number] => {
```

Comment:
is this the right type? `ReactNode`?

Reply:
I actually don't know the difference now that I think about it, I'll look it up

Reply:
Node is more permissive, like null, string number, and components, JSX.Element is just react components